python/rtypes/pcc/triggers.py

```diff
@@ -0,0 +1,49 @@
+
+
+class TriggerTime(object):
+    before = "b"
+    after = "a"
+
+
+class TriggerAction(object):
+    create = "c"
+    read = "r"
+    update = "u"
+    delete = "d"
+
+
+class TriggerProcedure(object):
+    def __init__(self, procedure, pcc_type, time, action, priority):
+        self.procedure = procedure
+        self.pcc_type = pcc_type
+        self.time = time
+        self.action = action
+        self.priority = priority 
+
+    def __call__(self, dataframe=None, new=None, old=None, current=None):
+        return self.procedure(dataframe, new, old, current)
+
+    def __eq__(self, v):
+        return v.priority == self.priority
+
+    def __lt__(self, v):
+        return v.priority < self.priority
```

Comment:
you can keep this like this, because < and > are operators that can have specific meaning outside equality.